### DISCUSSION TITLE
Dropdown Icon Not Showing

### DISCUSSION THREAD

**Discussion by Shrikant-28** [twbs/bootstrap#38940] [Q&A] [answered]
Before
![image](https://github.com/twbs/bootstrap/assets/59720723/e560c495-21bb-4b81-8a22-5d83aa275eb8)

After removing appearance css its showing dropdown icon
![image](https://github.com/twbs/bootstrap/assets/59720723/7450639a-1d7c-4382-9e84-e757eed24fc3)

**Comment by nkdas91** [comment, marked as answer]
Which version of Bootstrap are you using?

As of Bootstrap 5, the selects should have ```form-select``` class.
Please replace the class ```form-control``` with ```form-select``` on selects.

**Reply by Shrikant-28** [reply]
Thank you...Icon is showing now.
